packages/pn-pa-webapp/public/locales/it/notifiche.json

```diff
@@ -178,6 +178,7 @@
         "pec-error": "Indirizzo PEC non valido",
         "too-long-denomination-error-PF": "La lunghezza totale di nome e cognome non può superare gli 80 caratteri",
```

Comment:
@AndreaCimini90 this message should be "Inserisci max 80 caratter" (also for pg)

Reply:
we should use the same message for pf and pg